diff --git a/dist/api/generate-manifest-icons-entry.cjs b/dist/api/generate-manifest-icons-entry.cjs
index ffe352e2932849b21865e8a7b3b4beb127a18092..6b3f55b0949f8ca7268757964b65c06d6b72149a 100644
--- a/dist/api/generate-manifest-icons-entry.cjs
+++ b/dist/api/generate-manifest-icons-entry.cjs
@@ -6,7 +6,8 @@ function generateManifestIconsEntry(format, instruction) {
     icons.icons.push({
       src: icon.name,
       sizes: `${icon.width}x${icon.height}`,
-      type: icon.mimeType
+      type: icon.mimeType,
+      purpose: "any"
     });
   }
   for (const icon of Object.values(instruction.maskable)) {
diff --git a/dist/api/generate-manifest-icons-entry.mjs b/dist/api/generate-manifest-icons-entry.mjs
index 455d67ed66eb9f15d16bb1aaf9ef2c6bead18b0f..85918e903d2dd507866ae65305babf85ebb0b7e3 100644
--- a/dist/api/generate-manifest-icons-entry.mjs
+++ b/dist/api/generate-manifest-icons-entry.mjs
@@ -4,7 +4,8 @@ function generateManifestIconsEntry(format, instruction) {
     icons.icons.push({
       src: icon.name,
       sizes: `${icon.width}x${icon.height}`,
-      type: icon.mimeType
+      type: icon.mimeType,
+      purpose: "any"
     });
   }
   for (const icon of Object.values(instruction.maskable)) {
